api/src/pcapi/core/educational/factories.py

```diff
@@ -553,6 +553,11 @@ def create_reimbursed_stock(self, _create: bool, _extracted: typing.Any, **_kwar
         ReimbursedCollectiveBookingFactory.create(collectiveStock=stock)
 
 
+class ArchivedReimbursedCollectiveOfferFactory(ReimbursedCollectiveOfferFactory):
+    isActive = False
+    dateArchived = factory.LazyFunction(datetime.datetime.utcnow)
```

Comment:
Mettre une date dans le passé (histoire d'éviter l'ambiguité qui peut y avoir sur le jour même)

```suggestion
    dateArchived = factory.LazyFunction(datetime.datetime.utcnow - datetime.timedelta(days=7)
```

Reply:
Les 2 autres factory Archived ont utcnow, on les met aussi à jour pour - 7 jours ?